test-vectors/did_dht/resolve.json

```diff
@@ -1,6 +1,36 @@
 {
   "description": "did:dht resolution",
   "vectors": [
+    {
+      "description": "resolves successfully",
```

Comment:
to be clear, is this expected to hit a DHT resolver? or resolve locally?

Reply:
i would like it to hit a did resolver. this pr is related to https://github.com/TBD54566975/web5-go/pull/156 where i found that web5-go's did dht resolution fails this test.

Reply:
ah, i remember web5-js we mocked out all did dht calls. so yea, maybe it should resolve locally. either way, we need a test vector where one resolves successfully - current test cases are all failure ones from what i recall

i will turn this PR back to draft state and make sure web5-go works.

Reply:
fine if it hits a resolver, but it should not rely on a published DID. it should be a self contained test